storage/statedb/cache_hybrid.go

```diff
@@ -16,60 +16,82 @@
 
 package statedb
 
+import "github.com/go-redis/redis/v7"
+
 func NewHybridCache(config TrieNodeCacheConfig) (TrieNodeCache, error) {
 	redis, err := NewRedisCache(config)
 	if err != nil {
 		return nil, err
 	}
 
-	return &hybridCache{
+	return &HybridCache{
 		local:  NewFastCache(config),
 		remote: redis,
 	}, nil
 }
 
-// hybridCache integrates two kinds of caches: local, remote.
+// HybridCache integrates two kinds of caches: local, remote.
 // local cache uses memory of the local machine and remote cache uses memory of the remote machine.
-type hybridCache struct {
+type HybridCache struct {
 	local  TrieNodeCache
-	remote TrieNodeCache
+	remote *RedisCache
+	pubSub *redis.PubSub
```

Comment:
Why did you declare `pubSub`? You can access `pubSub` by `remote. pubSub`. 
Also, `pubSub` is not set.

Reply:
Oh, it was my mistake. Thanks!